### PR TITLE
Chore: add react-intl flow-typed

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by wenboyu2** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Before | After |
| ------------- |:----:|:------:|:-----:|
| Total | 0 B | 0 B | 0 B |

Compared against previously merged PR #254

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: .eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 flow/**
 lib/**
 dist/**
+flow-typed
```
Comment (priyajeet) [inline .eslintignore]: I think it needs to also be excluded in npmignore

Comment (DanDeMicco) [reply .eslintignore]: good idea, done in latest commit

**Pull request merged**